core/modules/syncer.js

```diff
@@ -281,6 +280,38 @@ Syncer.prototype.handleLazyLoadEvent = function(title) {
 	}
 };
 
+/* 
+Handle server update event
+*/
+Syncer.prototype.handleServerUpdate = function(tiddlerFields){
```

Comment:
The way that this method is written is expecting a skinny tiddler, is that what you want? Perhaps it would be more useful for the syncadaptor to invoke a method on the syncer that passes both the full tiddler fields and the tiddlerInfo?

Reply:
And yes, I extracted that code out of syncFromServer. It is correct. However, as I said, it is more likely that the sync adaptor would want to call storeTiddler.